red_blue_world/Quilt.py

```diff
@@ -13,13 +16,119 @@ class Quilt:
     def __init__(self):
         self._patches: Dict[PatchID, Patch] = {}
         self._active_patch_id: PatchID = (0, 0)
-        self._active_patch: Patch = ...
+        self._active_patch: Patch = self.build_patch()
+
+        self._t = 0
+        self._unload_clocks: Dict[PatchID, int] = {}
+
+        self._back_thread = ThreadPoolExecutor(max_workers=1)
```

Comment:
The background thread is mostly about latency instead of bandwidth (i.e. we should be able to offload work away from the hotpath, but there isn't really _that_ much work). So just one thread should be fine. We should stress test at a later time